commit 216ac69660ca5a1c9168ce986e69aad2c243781b (refs/changes/00/900/2)
Author: David Pacheco <dap@joyent.com>
Date:   2016-11-14T15:50:01-08:00 (2 years, 11 months ago)
    
    MANTA-3029 use git+https url in marlin README for "npm install"
    Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com>
    Approved by: Jordan Hendricks <jordan.hendricks@joyent.com>

diff --git a/README.md b/README.md
index c3e7047..33a068f 100644
--- a/README.md
+++ b/README.md
@@ -39,7 +39,7 @@ bits to build on OS X.  To use these tools:
 
 or just:
 
-    $ npm install git+ssh://git@github.com:joyent/manta-marlin.git
+    $ npm install git+https://github.com/joyent/manta-marlin.git
 
 At this point, you'll have several tools available inside the "sbin" directory:
 
